apps/studio/src/features/editing-experience/components/form-builder/components/LinkErrorBoundary.tsx

```diff
@@ -0,0 +1,50 @@
+import { PropsWithChildren } from "react"
+import { IconButton, Stack, Text } from "@chakra-ui/react"
+import { Infobox } from "@opengovsg/design-system-react"
+import { ErrorBoundary } from "react-error-boundary"
+import { BiTrash } from "react-icons/bi"
+
+interface LinkErrorBoundaryProps {
+  resetLink: () => void
+}
+export const LinkErrorBoundary = ({
+  resetLink,
+  children,
+}: PropsWithChildren<LinkErrorBoundaryProps>) => {
+  return (
+    <ErrorBoundary
+      fallbackRender={({ resetErrorBoundary }) => (
+        <Infobox
```

Comment:
question - should we also standardize to use Flex like in `BaseLinkControl` instead to have a more consistent code and UI?

https://github.com/opengovsg/isomer/blob/fc2128e2070ed16e69a81d71c686501d966aa592/apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/BaseLinkControl.tsx#L76-L83

Reply:
most of the time, yes - however i didn't do it here because we want the `Infobox` component from ogpds. the reason why i didn't use it last time was because (sadly enough) it [always](https://github.com/opengovsg/camp-design-system/blob/b9bddc4299d1b8c87801bb3635598f001019b3fb/packages/camp/src/Infobox/Infobox.tsx#L37) provides an icon